packages/editor-ui/src/components/SuggestedTemplates/SuggestedTemplatesPreviewModal.vue

```diff
@@ -55,9 +53,12 @@ function openCanvas() {
 			title: i18n.baseText('suggestedTemplates.notification.comingSoon.title'),
 			message: i18n.baseText('suggestedTemplates.notification.comingSoon.message'),
 			type: 'info',
+			duration: 10000,
 			onClick: showConfirmationMessage,
 		},
 	]);
+	localStorage.setItem(SUGGESTED_TEMPLATES_FLAG, 'false');
+	uiStore.deleteSuggestedTemplates();
```

Comment:
Do we have any tests for this feature? are we not adding any because it's a test?

Reply:
We do. See https://github.com/n8n-io/n8n/pull/8042/files. It seems we do not check for this specific behavior though.